Makefile

```diff
@@ -17,6 +17,7 @@ docker-build:
 	docker buildx create --name hp-image-builder --driver docker-container --bootstrap 2> /dev/null || true
 	docker buildx use hp-image-builder
 	docker buildx build \
+		--provenance=true --sbom=true \
```

Comment:
this is just to add the attestations to the image, to improve security and get a higher score
https://docs.docker.com/build/metadata/attestations/